pipeline-module/cookiecutter.json

```diff
@@ -1,5 +1,5 @@
 {
-  "handle": "SPHEREX-P-001",
+  "handle": "SSDC-MS-000",
```

Comment:
I think if this is a dedicated template for module specifications, we might only give the cookiecutter user the serial number to input rather than the whole handle.

Cookiecutter would let us then confirm the generated handle, right, just so the user would get confidence from seeing it?

Reply:
That was my original plan, but I ran into issues doing the zero-padded string formatting within the context of the Jinja template inside cookiecutter.json. This `handle` variable is a work-around to get something working quickly. I think I'll need to write a small Jinja extension to do the string formatting.

Reply:
What if we just reminded the user to type the leading zeros, for now?  This worked for Roman, the way you set up that template.